cmd/config/internal/commands/cmdcreatesubstitution_test.go

```diff
@@ -406,7 +427,7 @@ spec:
       - name: sidecar
         image: nginx::1.7.9 # {"$openapi":"my-image-subst"}
 `,
-			err: "cyclic substitution detected with name my-nested-subst",
+			err: "substitution with name my-nested-subst already exists",
```

Comment:
Based on the error, looks like this test no longer actually checks for cyclic substitutions? Would this still be possible with this change? If so, we should make sure we still check for it. If not, we should rename this test to reflect what is actually being tested.

Reply:
You are correct. This case should never occur with this check. I will remove the test.